spec/requests/media_spec.rb

```diff
@@ -167,7 +170,7 @@
         }
       end
 
-      it "updates the record" do
+      xit "updates the record", :skip_prefix_pool do
```

Comment:
@svogt0511 is this going to stay commented?

Reply:
Done.